app/scripts/lib/fxa-client.js

```diff
@@ -93,9 +85,11 @@ function (_, FxaClient, $, xhr, p, Session, AuthErrors, Constants) {
         verified: accountData.verified || false
       };
 
-      if (shouldFetchKeys(relier)) {
+      if (relier.isKeyFetchEnabled()) {
```

Comment:
`isKeyFetchEnabled()` is something I can imagine saying with a robotic voice. Maybe `.wantsKeys`? or `.shouldFetchKeys()`?